components/profile/activityTab/HistoryRow.vue

```diff
@@ -170,7 +168,9 @@ const interactionName = computed(
 
 const getAvatar = async () => {
   if (props.event.Item) {
-    avatar.value = await parseNftAvatar(props.event.Item)
+    avatar.value = (
+      await getNftMetadata(props.event.Item, urlPrefix.value)
+    ).image
```

Comment:
Please fix `parseNftAvatar` instead of fixing it here
as other components are also using `parseNftAvatar`